package.json

```
@@ -16,7 +16,7 @@
     "dependencies": {
         "@aws-sdk/client-s3": "^3.52.0",
         "@aws-sdk/client-ses": "^3.12.0",
-        "@cord.network/sdk": "0.8.4-beta.1",
+        "@cord.network/sdk": "0.8.0-3",
```

```suggestion
        "@cord.network/sdk": "^0.8.0-3",
```

to keep this future proof